starter-code/src/functions-and-arrays.js

```diff
@@ -74,27 +157,132 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+function howManyTimes(array, wordToCount){
+  if (array.length === 0) {
+  return false;
+}
+
+var count = 0;
+for (i = 0; i < array.length; i++) {
+  if (array[i] === wordToCount){
+    count++;
+  }
+}
+return count;
+}
```

Comment:
Genial, también podéis usar indexOf( ) o includes( ), echadle a cómo hacerlo con ambos